WoWsShipBuilder.ViewModels/ShipVm/DepthChargeDamageDistributionChartViewModelBase.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using ReactiveUI;
+using WoWsShipBuilder.Core.DataContainers;
+using WoWsShipBuilder.ViewModels.Base;
+
+namespace WoWsShipBuilder.ViewModels.ShipVm;
+
+public class DepthChargeDamageDistributionChartViewModelBase : ViewModelBase
```

Comment:
Rename to DepthChargeDamageDistributionViewModel, this is not a base viewmodel that gets extended with custom logic for each platform.
ViewModelBase classes should always be abstract.